css/at-rules.json

```diff
@@ -122,6 +122,43 @@
     "status": "nonstandard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@document"
   },
+  "@font-palette-values": {
+    "syntax": "@font-palette-values <dashed-ident> {\n  <declaration-list>\n}",
```

Comment:
`\n` seems to annoy the linter.

Reply:
Yes, they are not part of the syntax of syntax.

Reply:
```suggestion
    "syntax": "@font-palette-values <dashed-ident> {  <declaration-list> }",
```

Reply:
But why does other at-rules, e.g. `@font-face` right below, have `\n`? I thought it's for formatting.